src/executor/inplace_addto_detect_pass.cc

```diff
@@ -32,6 +32,7 @@ namespace mxnet {
 namespace exec {
 
 Graph DetectInplaceAddTo(Graph g) {
+  size_t enable_inplace = dmlc::GetEnv("NNVM_EXEC_ENABLE_INPLACE", true);
```

Comment:
Why not use `MXNET_EXEC_ENABLE_INPLACE ` since the pass is in MXNet?

Reply:
@piiswrong